pandas_datareader/data.py

```diff
@@ -56,6 +66,42 @@ def get_quote_google(*args, **kwargs):
     return GoogleQuotesReader(*args, **kwargs).read()
 
 
+def get_tops_iex(*args, **kwargs):
+    return IEXTops(*args, **kwargs).read()
```

Comment:
Agreed, no need to pollute the namespace further even for consistency's sake. Pushed an update with this change and docstrings, but I left imports in for the functions implemented by DataReader.